common-moonmage.lic

```diff
@@ -35,6 +36,7 @@ module DRCMM
   end
 
   def store_telescope?(telescope_name = "telescope", storage)
+    return true unless DRCI.in_hands?(telescope_name);
```

Comment:
Stylistically, we haven't been using `;`

Reply:
haha, that's muscle reflex from me being in a different programming language all day that needed them 😆 
```suggestion
    return true unless DRCI.in_hands?(telescope_name)
```